discord/message.py

```diff
@@ -1114,3 +1140,43 @@ async def ack(self):
         if state.is_bot:
             raise ClientException('Must not be a bot account to ack messages.')
         return await state.http.ack_message(self.channel.id, self.id)
+
+    async def reply(self, content=None, **kwargs):
```

Comment:
`Context.reply` has a `mention_author` bool why doesn't this? (ties in with my above comment)